flink-connector-hbase-1.4/src/test/java/org/apache/flink/connector/hbase1/HBaseDynamicTableFactoryTest.java

```diff
@@ -256,6 +256,19 @@ public void testBufferFlushOptions() {
         assertEquals(expected, actual);
     }
 
+    @Test
+    public void testSinkIgnoreNullValueOptions() {
+        Map<String, String> options = getAllOptions();
+        options.put("sink.ignore-null-value", "true");
+
+        ResolvedSchema schema = ResolvedSchema.of(Column.physical(ROWKEY, STRING()));
+
+        DynamicTableSink sink = createTableSink(schema, options);
+        HBaseWriteOptions expected = HBaseWriteOptions.builder().setIgnoreNullValue(true).build();
+        HBaseWriteOptions actual = ((HBaseDynamicTableSink) sink).getWriteOptions();
+        assertEquals(expected.isIgnoreNullValue(), actual.isIgnoreNullValue());
```

Comment:
I think instantiating a dummy expected object can be avoided, we can simply use `assertThat(actual.isIgnoreNullValue()).isTrue()`.